test/reductions/thomasvandoren/scanCountsInherit.chpl

```diff
@@ -8,7 +8,7 @@
  * could determine a ranking of the particles within each octant.
  */
 
-public use counts;
+public use CountsMod;
```

Comment:
Renamed the module to avoid conflict between `module counts` and `class counts`

Reply:
This one gives me pause since I think we've both said that we want to be able to continue supporting these cases, making me want to resolve it in some other way (either a change to the compiler or a change to the use of counts.  As mentioned on chat, I think my preference would be to have `public use counts` not bring in the `module counts` if we could make that work.  Otherwise, I'd probably rename either the counts module or the counts type in the `use` statement rather than renaming the module itself.

Reply:
Or, even easier: I don't think there's anything about this `use` that requires it to be `public`?  So I'd just remove the `public` rather than renaming the module

(though I still believe the previous thoughts as well; though we've got other tests/cases that exhibit this pattern I think, so no need for this one to be the exemplar of it (?)).

Reply:
Yes it seems to work to change this one to `use counts` and so I will do that here.